### PR TITLE
Remove CloudFront CDN upload on version release.

### LINKED ISSUE #706: Remove CDN upload functionality
We added a feature a few months back that would compile OUI's CSS and upload it to a CloudFront CDN. I'd like to remove this functionality to simplify OUI's deployment process since it is no longer needed.

### PR DESCRIPTION
**Pull request by danoc**
We used to upload a compiled version of our CSS to `oui.cdn.optimizely.com` but I'd like to remove this since it is no longer needed and the code that supports it is confusing/hard to debug.

This fixes #706.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,12 +54,11 @@
   },
   "scripts": {
     "test": "npm run eslint && npm run jest && codecov && npm run scss-lint && npm run icons:update && npm run docs:build",
-    "start": "bundle install && bundle update && npm install && npm run sass && npm run icons:update && npm run babel && npm run docs:build",
+    "start": "bundle install && bundle update && npm install && npm run icons:update && npm run babel && npm run docs:build",
     "jest": "jest --config .jest.config.json",
     "jest:watch": "jest --config .jest.config.json --watch",
     "eslint": "eslint --ext .jsx --ext .js . --ignore-path .gitignore --max-warnings 0",
     "scss-lint": "bundle exec scss-lint src/",
-    "sass": "mkdir -p dist/css && node-sass src/oui/oui.scss | postcss --use autoprefixer > dist/css/oui.css",
```
Comment (spencerwilson-optimizely) [inline package.json]: Do the `postcss` and `autoprefixer` here correspond to the two similarly-named dependencies that are removed above? Do you still need `postcss-loader`?

File: .travis.yml [hunk 2]
```diff
@@ -13,27 +13,12 @@ before_deploy:
...
 # https://oncletom.io/2016/travis-ssh-deploy/
 - eval "$(ssh-agent -s)"
 - chmod 600 $TRAVIS_BUILD_DIR/deploy_rsa
 - ssh-add $TRAVIS_BUILD_DIR/deploy_rsa
 deploy:
-- provider: s3
-  access_key_id:
-    secure: hUfpx4qlgXNxuq7fXCCI9BpbJ1aSQJFy58CtuDW077i3WULaoKJ5aTuVOPDQY3V+hgskj5ECWXG7i7o0ZiVAyBLR92IKJQiT5NvSWkEyw0WJ/L0b2QkVB9lx1sX2wzg6k03inJnPoUIYAkhAdtj1W6sMmLXbgOouhT7bsuYUi9M=
-  secret_access_key:
-    secure: sHcezDuWh9s341ruDx4pDMmjDWuoQ7cf5jhUQu3YyNQ8zaPRfVLztnYcJFZCUAznJi7Sswb3hIVvEQq+PXgd2YibdI6lXVrFff/cqbpRT9VBllIG9gRQbXcTJ22u+SzdB9ikE4wKqqZx2aUX1GpDIQc0TigsFmXhxNxKG8gkSik=
```
Comment (spencerwilson-optimizely) [inline .travis.yml]: This is a public repo; are these secret keys? :O

File: .travis.yml [hunk 3]
```diff
@@ -13,27 +13,12 @@ before_deploy:
...
-  bucket: optimizely-oui
-  local-dir: s3
-  region: us-west-2
-  acl: public-read
-  on:
-    repo: optimizely/oui
-    tags: true
 - provider: npm
   email:
     secure: Y+A/HOGTYtASn+cHBFxslq52Ij8sdS5I+uvKiZF6466fwJytlGzdyR9e+4NGkul0UgJl3CUY5xKZ7LhyNBPxQUaG9DmlX8pkDiuHlcFlMbCtw+aTLhRS+pfpGbOlKZUWZcM5jZF+TvPIEqN/q5/p5kZnIqp9dmVijOSu7/t0zxA=
```
Comment (spencerwilson-optimizely) [inline .travis.yml]: What is this?

File: .travis.yml (hunk 2 above)
Comment (danoc) [inline .travis.yml]: This is encrypted with Travis-CI:
https://docs.travis-ci.com/user/encryption-keys/

File: .travis.yml (hunk 3 above)
Comment (danoc) [inline .travis.yml]: This is an encrypted token we use to release a new version of OUI on NPM.

File: package.json (hunk 1 above)
Comment (danoc) [inline package.json]: I tried removing `postcss-loader` but realized that I need it in webpack to add vendor prefixes to our CSS in the OUI documentation.

**Pull request merged**